server/sonar-server/src/main/java/org/sonar/server/app/ServerProcessLogging.java

```diff
@@ -88,10 +88,9 @@ public LoggerContext configure(Props props) {
     LoggerContext ctx = helper.getRootContext();
     ctx.reset();
 
-    helper.enableJulChangePropagation(ctx);
-
     configureRootLogger(props);
     helper.apply(logLevelConfig, props);
+    helper.enableJulChangePropagation(ctx);
```

Comment:
what is the reason behind moving call to enableJulChangePropagation after configuring the loggers in SLF4J?